mitosheet/src/components/endo/ColumnHeader.tsx

```diff
@@ -94,6 +96,24 @@ const ColumnHeader = (props: {
         setTimeout(() => focusGrid(props.containerRef.current), 100);
     }
 
+    const getColumnIndexesFullWidthArray = (columnIndexes: number[]): WidthData[]  => {
```

Comment:
can we rename to `getWidthArrayAtFullWidthForColumnIndexes`

Reply:
Also, I think this can go in `widthUtils.tsx` -- it doesn't need to be defined in ColumnHeader for any reason.